docs/getting-started/release_package.md

```diff
@@ -77,7 +77,12 @@
       # verify the configurations before running cloud(edgecontroller)
```

Comment:
the changes to release_package.md for cri should be part of a separate PR. This is the design PR.

Reply:
Removed